### PR TITLE
paddle_take_along_axis

### LINKED ISSUE #21499: take_along_axis

### PR DESCRIPTION
**Pull request by ManoBharathi93**
Close #21499

### CODE REVIEW

**Comment by Daniel4078** [comment]
ivy-gardener

File: ivy/functional/backends/paddle/experimental/manipulation.py [hunk 1]
```diff
@@ -403,6 +404,10 @@ def atleast_3d(
     {"2.5.1 and below": {"cpu": ("int8",)}},
     backend_version,
 )
+@with_supported_device_and_dtypes(
```
Comment (Daniel4078) [inline ivy/functional/backends/paddle/experimental/manipulation.py:407]: Why is this part added? I though it is not really related?

**Review by Daniel4078 — APPROVED** [review]
Looks good to me!

**Comment by ManoBharathi93** [comment]
> Looks good to me!

Thank you daniel
